protos/sources/vega/checkpoint/v1/checkpoint.proto

```diff
@@ -213,69 +213,87 @@ message MarketTracker {
 message MarketActivityTracker {
   string market = 1;
   string asset = 2;
-  // To be deprecated
   repeated PartyFees maker_fees_received = 3;
   repeated PartyFees maker_fees_paid = 4;
   repeated PartyFees lp_fees = 5;
-  // end deprecation
   string proposer = 6;
   repeated string bonus_paid = 7;
   string value_traded = 8;
   bool ready_to_delete = 9;
 
-  repeated PartyFeeData maker_fees_received_history = 10;
-  repeated PartyFeeData maker_fees_paid_history = 11;
-  repeated PartyFeeData lp_fees_history = 12;
-  repeated TWPositionData time_weighted_position_history = 13;
-  repeated ReturnsData returns_data = 14;
-  repeated TWNotionalPosition time_weighted_notional_position = 15;
+  repeated TWPositionData time_weighted_position = 10;
+  repeated TWNotionalData time_weighted_notional = 11;
+  repeated ReturnsData returns_data = 12;
 
-  DataWithHistory total_maker_fees_received = 16;
-  DataWithHistory total_maker_fees_paid = 17;
-  DataWithHistory total_lp_fees = 18;
+  repeated EpochPartyFees maker_fees_received_history = 13;
+  repeated EpochPartyFees maker_fees_paid_history = 14;
+  repeated EpochPartyFees lp_fees_history = 15;
+
+  repeated EpochTimeWeightPositionData time_weighted_position_data_history = 16;
```

Comment:
Should this be EpochTimeWeight**ed**PositionData?

Reply:
That huge list of changes appeared when I messed up my rebase; that change is actually already committed